### PR TITLE
Fix header and footer component loading

### PR DESCRIPTION
**Pull request by rlee80**
Moves the loading of the header and footer components from every parent component to just the main app component.

- Link to live demo: http://pr-239-evalai.surge.sh

Header:
image

Footer:
image

### CODE REVIEW

File: src/app/components/challenge/challenge.component.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<app-header-static></app-header-static>
+
```
Comment (Ram81) [inline src/app/components/challenge/challenge.component.html]: @rlee7 Can you remove the blankline

File: src/app/components/contact/contact.component.html [hunk 2]
```diff
@@ -1,4 +1,4 @@
-<app-header-static></app-header-static>
+
```
Comment (Ram81) [inline src/app/components/contact/contact.component.html]: Same here

File: src/app/components/dashboard/dashboard.component.html [hunk 3]
```diff
@@ -1,14 +1,12 @@
 <app-side-bar *ngIf="authService.isAuth"></app-side-bar>
-<app-header-static></app-header-static>
+
```
Comment (Ram81) [inline src/app/components/dashboard/dashboard.component.html]: Same here

File: src/app/components/get-involved/get-involved.component.html [hunk 4]
```diff
@@ -1,4 +1,4 @@
-<app-header-static></app-header-static>
+
```
Comment (Ram81) [inline src/app/components/get-involved/get-involved.component.html]: Same here

File: src/app/components/home/home.component.html [hunk 5]
```diff
@@ -243,4 +243,4 @@ <h4 class="w-300 center">Subscribe to our newsletters</h4>
   <!--landing-footer></landing-footer-->
 </div>
 
-<app-footer></app-footer>
+
```
Comment (Ram81) [inline src/app/components/home/home.component.html]: Same here

File: src/app/components/home/home.component.html [hunk 6]
```diff
@@ -1,4 +1,4 @@
-<app-header-static></app-header-static>
+
```
Comment (Ram81) [inline src/app/components/home/home.component.html]: Same here

File: src/app/components/publiclists/publiclists.component.html [hunk 7]
```diff
@@ -1,13 +1,10 @@
 <app-side-bar *ngIf="authService.isAuth"></app-side-bar>
-<app-header-static></app-header-static>
+
```
Comment (Ram81) [inline src/app/components/publiclists/publiclists.component.html]: Same here

File: src/app/components/profile/profile.component.html [hunk 8]
```diff
@@ -1,4 +1,4 @@
-<app-header-static></app-header-static>
+
```
Comment (Ram81) [inline src/app/components/profile/profile.component.html]: Same here

**Review by Ram81 — CHANGES_REQUESTED** [review]

**Review by Ram81 — APPROVED** [review]
@rlee7 LGTM 👍 . @Sanji515 @lunayach can you guys review once.

**Comment by Sanji515** [comment]
Screenshot from 2019-12-15 11-33-59


@rlee7 Can you please fix this (^)

**Comment by rlee80** [comment]
@Sanji515 What page is that? I can't seem to replicate on my machine

**Comment by Sanji515** [comment]
> @Sanji515 What page is that? I can't seem to replicate on my machine

You can see this on the any page where this sidebar displays. Like `Dashboard page`.

**Comment by lunayach** [comment]
@rlee7 I'm also seeing the same issue.

**Comment by rlee80** [comment]
@Sanji515 I think it should be fixed now, please confirm for your machine

**Comment by pushkalkatara** [comment]
1. The `All Challenges` and `Forum` tabs are displayed on the login page. Please fix this.
bug

2. Still, I think the dashboard issue is not fixed.
bug 2


Also, please update the branch with the base branch.

**Pull request closed without merging**